### PR TITLE
refine NextUI article demo app

### PR DESCRIPTION
**Pull request by nibble0101**
This PR adds the demo app for the refine NextUI article.

### CODE REVIEW

File: examples/blog-refine-nextui/src/components/modal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  Button,
+} from "@nextui-org/react";
+
+import { DeleteModalPropsI } from "../../interfaces";
+
+export const DeleteModal: React.FC<DeleteModalPropsI> = ({
```
Comment (alicanerdurmaz) [inline examples/blog-refine-nextui/src/components/modal/index.tsx:13]: is this typo ?

```suggestion
export const DeleteModal: React.FC<IDeleteModalProps> = ({
```

File: examples/blog-refine-nextui/src/components/table/RecentSalesTable.tsx [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+            </DropdownTrigger>
+            <DropdownMenu
+              aria-label="Select number of rows per page"
+              variant="flat"
+              color="primary"
+              disallowEmptySelection
+              selectionMode="single"
+              selectedKeys={[`${pageSize}`]}
+              onSelectionChange={(rows) => {
+                console.log(+(rows as Set<string>).values().next().value);
```
Comment (alicanerdurmaz) [inline examples/blog-refine-nextui/src/components/table/RecentSalesTable.tsx:196]: Is this left on purpose?

```suggestion
                
```

File: examples/blog-refine-nextui/src/pages/products/edit.tsx [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+                  />
+                );
+              }}
+            />
+            <Controller
+              name="category.id"
+              control={control}
+              rules={{ required: "Category is required." }}
+              render={({ field, fieldState }) => {
+                console.log("Inside controller", field);
```
Comment (alicanerdurmaz) [inline examples/blog-refine-nextui/src/pages/products/edit.tsx:106]: Is this left on purpose?

```suggestion
                console.log("Inside controller", field);
```

File: examples/blog-refine-nextui/src/pages/products/list.tsx [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+              </DropdownTrigger>
+              <DropdownMenu
+                aria-label="Select number of rows per page"
+                variant="flat"
+                color="primary"
+                disallowEmptySelection
+                selectionMode="single"
+                selectedKeys={[`${pageSize}`]}
+                onSelectionChange={(rows) => {
+                  console.log(+(rows as Set<string>).values().next().value);
```
Comment (alicanerdurmaz) [inline examples/blog-refine-nextui/src/pages/products/list.tsx:239]: Is this left on purpose?
```suggestion
                  console.log(+(rows as Set<string>).values().next().value);
```

**Review by alicanerdurmaz — CHANGES_REQUESTED** [review]

File: examples/blog-refine-nextui/src/components/modal/index.tsx (hunk 1 above)
Comment (nibble0101) [reply examples/blog-refine-nextui/src/components/modal/index.tsx]: I've made the requested changes.

**Review by alicanerdurmaz — APPROVED** [review]

**Pull request merged**